aggregate/PUMS/pums_0812_1519_economics.py

```diff
@@ -0,0 +1,228 @@
+import pandas as pd
+from aggregate.aggregation_helpers import order_aggregated_columns
+from internal_review.set_internal_review_file import set_internal_review_files
+from utils.PUMA_helpers import borough_name_mapper, clean_PUMAs, get_all_boroughs
+from utils.dcp_population_excel_helpers import (
+    race_suffix_mapper_global,
+    count_suffix_mapper_global,
+    median_suffix_mapper_global,
+    remove_duplicate_cols,
+)
+
+occupations = ["mbsa", "srvc", "slsoff", "cstmnt", "prdtrn"]
+education_levels = ["lths", "hs", "smcol", "bchpl"]
+industries = [
+    "agff",
+    "cnstn",
+    "mnfct",
+    "whlsl",
+    "rtl",
+    "trwhu",
+    "info",
+    "fire",
+    "pfmg",
+    "edhlt",
+    "arten",
+    "oth",
+    "pbadm",
+]
+ages = ["p25p", "p16t64"]
```

Comment:
Hey Sasha looking at the [data dictionary](https://nyco365.sharepoint.com/:x:/s/NYCPLANNING/civic-engagement-studio/EVBcBM4HnJFCtab3bRKNOxgBSTVR9j0wlk-D4uM1GxyNhw?e=NpCgRN). Should the age_p25p be age_p25pl?

Reply:
good catch, can change that now